### PR TITLE
Adding ImportInterface for Zend\Feed\Reader\Reader

### PR DESCRIPTION
**Pull request by wesleywillians**
For people who want to use Reader as only a component.

### CODE REVIEW

File: library/Zend/Feed/Reader/ReaderImportInterface.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Feed\Reader;
+
+
+interface ReaderImportInterface
+{
+    public static function import($uri, $etag = null, $lastModified = null);
```
Comment (Ocramius) [inline library/Zend/Feed/Reader/ReaderImportInterface.php]: This class should include detailed and descriptive docblocks

File: library/Zend/Feed/Reader/ReaderImportInterface.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+     * Imports a feed from a file located at $filename.
+     *
+     * @param  string $filename
+     * @throws Exception\RuntimeException
+     * @return Feed\FeedInterface
+     */
+    public static function importFile($filename);
+
+
+}
```
Comment (samsonasik) [inline library/Zend/Feed/Reader/ReaderImportInterface.php]: add new blank line after <code>}</code> for eof_ending.

**Comment by samsonasik** [comment]
travis build failure : 
```
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error

1) Reader/ReaderImportInterface.php (braces, eof_ending)
```
it can easily be fixed by php-cs-fixer : 
```
vendor/bin/php-cs-fixer fix library/Zend/Feed/Reader/ReaderImportInterface.php --fixers=braces,eof_ending
```

**Comment by wesleywillians** [comment]
Thanks for the feedback. I have just commited with: php-cs-fixer

**Comment by weierophinney** [comment]
Merged to develop for release with 2.4.

**Pull request closed without merging**